libraries/anvill_passes/src/LowerRemillMemoryAccessIntrinsics.cpp

```diff
@@ -66,7 +68,9 @@ static void ReplaceMemWriteOp(llvm::CallBase *call_inst, llvm::Type *val_type) {
     val = ir.CreateFPExt(val, val_type);
   }
 
-  (void) ir.CreateStore(val, ptr);
+  val = ir.CreateStore(val, ptr);
```

Comment:
My understanding is that the calls to `ir` above this line are also generating instructions. Would you expect those to also be tagged with this same metadata?

Reply:
Yes, these new instructions should be tagged with the metadata of the `CallInst` which they replace.